NOMENCLATURE.md

```diff
@@ -1,69 +1,69 @@
 # Nomenclature
 
-To keep track of the various software components in Swarm, this document
-defines various aspects of the Swarm system, as referenced in this code base.
+To keep track of the software components in Swarm, this document defines
+various aspects of the Swarm system as referenced in _this_ code base.
 
 Several of these definitions may be a part of the product, while others are
 simply for communicating about backend components. Where this distinction is
 important, it will be called out.
 
 ## Overview
 
-There are several moving parts in a swarm cluster. This section attempts to
+There are several moving parts in a Swarm cluster. This section attempts to
```

Comment:
Yes. but hold off. it's the same for a lot of nouns in the doc. so give me a few mins.